pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -294,6 +295,8 @@ func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha2.ZoneLi
 		nrtNew := v1alpha2.NodeResourceTopology{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:            w.nodeName,
+				Namespace:       w.kubernetesNamespace,
+				Labels:          map[string]string{nfdgc.OwnerPod: os.Getenv("POD_NAME")},
```

Comment:
I think we should include the namespace in the label (and then check it in nfd-gc)

```suggestion
				Labels:          map[string]string{nfdgc.OwnerPod: fmt.Sprintf("%s/%s", w.kubernetesNamespace, os.Getenv("POD_NAME"))},
```